[UPDATE]Add comment

ChaeyeonSeo opened the pull request:
I add comments and move comments to be in appropriate location.

ChaeyeonSeo committed:
[UPDATE]add coments

ChaeyeonSeo committed:
[UPDATE]fix coments

ChaeyeonSeo committed:
[UPDATE]update comments

ChaeyeonSeo committed:
[UPDATE]add comment

devquint committed:
[UPDATE] commenting convention

one space added between # and actual comment

devquint committed:
[UPDATE] commenting convention

same as above, one space between # and comment

devquint reviewed (approved):
I fixed a little bit about commenting convention.

Hyuntae-Jeong committed:
Update dictionaries.py

changed position of comment.
placed comment after declaring person['last]

Hyuntae-Jeong commented:
Fixed position of comment if dictionaries.py

devquint reviewed (changes requested):
Commenting convention for this project is writing commenting above the code.
```python
# Description for below
some_code(...)
```
What about reverting latest commit? 

Or you may comment like (bad but simple example)
```python
# After the below code run, a = 5
a=5;
```

Hyuntae-Jeong committed:
[Update] Update dictionaries.py

i've changed comment  # person = {'last': 'Harrison', 'first': 'Christopher'} into # person = {'first': 'Christopher', 'last': 'Harrison'} in previous commit. But i've restored it since I'm not sure about the order.
Please check the order that I'm confused about.

Hyuntae-Jeong commented on python-for-beginners/11 - Collections/dictionaries.py:
```diff
@@ -1,4 +1,7 @@
 person = {'first': 'Christopher'}
+
 person['last'] = 'Harrison'
+# person = {'last': 'Harrison', 'first': 'Christopher'}
```
I'm not sure if 'first' comes first or 'last' comes first

devquint replied:
I thinks it just depend upon you.
Python dictionary is unordered, so there's no rule.

Hyuntae-Jeong replied:
Oh cool. I was confused about that.
I will not change the order but add comment about python dictionary
Thanks

Hyuntae-Jeong committed:
[Update] Added additional comment about dictionary

added description of python dictionary that the orders don't matter

ChaeyeonSeo commented:
I think you don't get "Commenting convention for this project is writing commenting above the code." correctly.

ChaeyeonSeo committed:
[UPDATE]update comments

ChaeyeonSeo committed:
[UPDATE]add comments

ChaeyeonSeo committed:
[UPDATE]add comments-ranges.py
